### DISCUSSION TITLE
Linux desktop framwork

### DISCUSSION THREAD

**Discussion by chis-optimus** [dotnet/wpf#9448] [Q&A]
Dotnet has already reached its 9th version. When will we finally get a desktop development framework that truly supports Linux?

**Comment by miloush** [comment]
Duplicate of #48 (and #3952 and #256)

**Reply by lextm** [reply]
Not exactly duplicate when those threads all ended up with advertisements for a closed source fork (and then locked).

There are open source community WPF forks (>2) coming soon to fill the gaps, so some WPF community roadmap work is needed when Microsoft invests so limited in this vast ecosystem.

**Comment by chis-optimus** [comment]
Dear Sir or Miss.
This is an e-mail from Johnnie' QQ Mail.Thank you for your E-mail.I apologize that I am unable to respond to you immediately . I will get back to you as soon as I return.

**Comment by chis-optimus** [comment]
Dear Sir or Miss.
This is an e-mail from Johnnie' QQ Mail.Thank you for your E-mail.I apologize that I am unable to respond to you immediately . I will get back to you as soon as I return.

**Reply by lindexi** [reply]
@chis-optimus Please check your e-mail settings.
